noise.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title></title>
+    <link href="noise.css" rel="stylesheet" />
+  </head>
+  <body>
+    <button id="record">Record</button>
```

Comment:
Based on the javascript file it appears that record functionality isn't set up yet, but I really like that idea!